source/user/scf.rst

```diff
@@ -53,7 +53,7 @@ as
 
 .. math::
 
-    \mathbf{F} = \mathbf{T} + \mathbf{V} + \mathbf{J} + \mathbf{K}\;
+    \mathbf{F} = \mathbf{T} + \mathbf{V} + \mathbf{J} - \mathbf{K}\;
```

Comment:
This depends on how K is defined. In the current form the minus sign is included, and the form also works for DFT where K is evaluated by quadrature.

Reply:
Thanks, that makes sense. I have changed it back.